### ISSUE TITLE
_init_default initializer

### ISSUE THREAD

**Issue by kasiabozek** [dmlc/MXNet.jl#45]
The '''_init_default''' function doesn't exist in the Initializer. The call was made when initializer was called for _logisticregressionoutput0_label_ which I'm not sure should happen.

**Comment by pluskid** [comment]
The `_init_default` method should be a fallback that raise an error (I just added it). It seems strange that the label needs to be initialized. Can you post the network architecture you defined here?

**Comment by kasiabozek** [comment]
```julia
data = mx.Variable(:data)
label = mx.Variable(:label_bee)

conv1 = @mx.chain mx.Convolution(data=data, kernel=(3,3), pad=(1,1), num_filter=n_filter[1], workspace=1024)  =>
                  mx.Activation(act_type=:relu) =>
                  mx.Convolution(kernel=(3,3), pad=(1,1), num_filter=n_filter[2], workspace=1024)  =>
                  mx.Activation(act_type=:relu) =>
                  mx.Pooling(pool_type=:max, kernel=(2,2), stride=(2,2))

fc1 = @mx.chain mx.Flatten(data=conv1) =>
                mx.FullyConnected(num_hidden=fc_out_dim) =>
                mx.Activation(act_type=:sigmoid) =>
                mx.FullyConnected(num_hidden=1) => 
                mx.Activation(act_type=:sigmoid)
loss = mx.LogisticRegressionOutput(data=fc1, label=label)

net = loss
```
